testkit/tests/system_api.rs

```diff
@@ -52,3 +52,14 @@ fn test_network() {
     assert_eq!(info.protocol_version, PROTOCOL_MAJOR_VERSION);
     assert!(info.services.is_empty());
 }
+
+#[test]
+fn test_consensus_status() {
+    let testkit = TestKitBuilder::validator().with_validators(2).create();
```

Comment:
I propose to change the number of validators to 1 in order to make the test more meaningful.